database/migrations/2021_04_10_195515_add_permission_restriction_override.php

```diff
@@ -0,0 +1,29 @@
+<?php
```

Comment:
Could I trouble you to do this (add a migration) for the other permissions & roles as well <3

Reply:
it would break all the tests. the best thing would be to just replace this migration with a seeder and remove the seeder form all the tests. Is that alright @alexstojda ?

Reply:
Yeah if you have time :) Else its ok

Reply:
@alexstojda I did the things